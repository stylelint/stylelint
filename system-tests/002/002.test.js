'use strict';

const stylelint = require('../../lib');
const systemTestUtils = require('../systemTestUtils');

it('002', () => {
	return stylelint
		.lint({
			files: [systemTestUtils.caseStylesheetGlob('002')],
			configFile: systemTestUtils.caseConfig('002'),
		})
		.then((output) => {
			expect(systemTestUtils.prepResults(output.results)).toMatchSnapshot();
		});
});
