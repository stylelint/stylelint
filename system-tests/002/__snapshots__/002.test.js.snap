// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`002 1`] = `
Array [
  Object {
    "deprecations": Array [
      Object {
        "reference": "https://stylelint.io/user-guide/rules/declaration-block-no-ignored-properties/",
        "text": "'declaration-block-no-ignored-properties' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/media-feature-no-missing-punctuation/",
        "text": "'media-feature-no-missing-punctuation' has been deprecated and in 8.0 will be removed.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/rule-nested-empty-line-before/",
        "text": "'rule-nested-empty-line-before' has been deprecated and in 8.0 will be removed. Instead use 'rule-empty-line-before'.",
      },
      Object {
        "reference": "https://stylelint.io/user-guide/rules/rule-non-nested-empty-line-before/",
        "text": "'rule-non-nested-empty-line-before' has been deprecated and in 8.0 will be removed. Instead use 'rule-empty-line-before'.",
      },
    ],
    "errored": undefined,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "source": "002/stylesheet.css",
    "warnings": Array [],
  },
]
`;
